integration-tests/ci-visibility.spec.js

```diff
@@ -67,6 +67,7 @@ const testFrameworks = [
     ...mochaCommonOptions,
     name: 'mocha',
     testFile: 'ci-visibility/run-mocha.mjs',
+    dependencies: ['@istanbuljs/esm-loader-hook'],
```

Comment:
don't you need mocha, and the rest of dependencies? The tests are failing right now

Reply:
Makes sense, I thought the object would merge the two arrays in the same field